src/pilot/assign_role.py

```diff
@@ -428,9 +459,11 @@ def define_storage_logical_disks(drac_client, raid_controller_name):
     #
     # A successful call returns a list, which may be empty; otherwise,
     # None is returned.
-    jbod_capable = drac_client.is_jbod_capable(raid_controller_name)
+    if not raid_controller:
+        raid_controller = boss_controller
```

Comment:
This section of code is about defining jbod disks for the remaining disks controlled by the RAID controller that are not being used for the RAID1 to install the OS on.  As it's written, in the BOSS+HBA330 case, it will ask the BOSS if it's JBOD capable below, then call into define_jbod_logical disks below with no remaining_physical_disks.  Instead of doing this, I would rather we just change line 462 above to "if raid_controller:" and delete line 463, then put lines 464-467 inside that "if" statement.  That way, if there is a BOSS+HBA, this code won't be executed, which is what I would expect.

Reply:
We know that the BOSS controller can only control 2 disks, and both of those disks will be used for the OS RAID1, so there is no need to execute this code if a BOSS is present since it will have no other disks to create JBOD RAIDs with.

Reply:
Agreed Chris. I have updated the code which defining jbod disks for the remaining disks.